templates/back/signalement/view/edit-modals/edit-coordonnees-foyer.html.twig

```diff
@@ -11,55 +14,77 @@
                                     <option value="{{ enum('App\\Entity\\Enum\\ProprioType').PARTICULIER.name }}" {{ signalement.typeProprio is same as enum('App\\Entity\\Enum\\ProprioType').PARTICULIER ? 'selected' : '' }}>{{ enum('App\\Entity\\Enum\\ProprioType').PARTICULIER.label }}</option>
                                 </select>
                             </div>
                             <div class="fr-input-group">
-                                <label class="fr-label" for="nomStructure">Nom de la structure</label>
-                                <input class="fr-input" type="text" name="nomStructure" value="{{ signalement.structureDeclarant }}">
+                                <label class="fr-label" for="nomStructure">Nom de la structure </label>
+                                <input class="fr-input" type="text" id="nomStructure" name="nomStructure" value="{{ signalement.structureDeclarant }}">
                             </div>
                             {% endif %}
 
                             <div class="fr-select-group">
-                                <label class="fr-label" for="civilite">Civilité</label>
-                                <select class="fr-select" name="civilite">
+                                <label class="fr-label" for="civilite">Civilité {{ show_label_facultatif(
+                                        'App\\Dto\\Request\\Signalement\\CoordonneesFoyerRequest',
+                                        'civilite',
+                                        signalement.profileDeclarant)  }}
+                                </label>
+                                <select class="fr-select" id="civilite" name="civilite">
                                     <option value="" {{ signalement.civiliteOccupant not in ['mme', 'mr'] ? 'selected' : '' }}></option>
                                     <option value="mme" {{ signalement.civiliteOccupant is same as 'mme' ? 'selected' : '' }}>Madame</option>
                                     <option value="mr" {{ signalement.civiliteOccupant is same as 'mr' ? 'selected' : '' }}>Monsieur</option>
                                 </select>
                             </div>
 
                             <div class="fr-input-group">
-                                <label class="fr-label" for="nom">Nom</label>
-                                <input class="fr-input" type="text" name="nom" value="{{ signalement.nomOccupant }}" maxlength="50">
+                                <label class="fr-label" for="nom">Nom {{ show_label_facultatif(
+                                        'App\\Dto\\Request\\Signalement\\CoordonneesFoyerRequest',
+                                        'nom',
+                                        signalement.profileDeclarant)  }}</label>
+                                <input class="fr-input" type="text" id="nom" name="nom" value="{{ signalement.nomOccupant }}" maxlength="50">

```

Comment:
ça fait des doublons d'`id` (avec les autres modales), ce n'est pas valide en html
et du coup l'attribut `for` ne va pas forcément pointer vers le bon
il faut des `id` vraiment uniques

Reply:
Effectivement :-) dire que j'avais hésité à me lancer la dedans :-D

Reply:
Ils ont tous des id différents maintenant